web/packages/teleport/src/services/resources/resource.ts

```diff
@@ -198,3 +212,39 @@ class ResourceService {
 }
 
 export default ResourceService;
+
+export async function fetchRole(
```

Comment:
idk if it matters how we define these functions, b/c I see a mixture of

```js
// wrap it in an object
export const someServiceName = {
    fn1(),
    fn2(),
}

// or define them standalone
export fn1()
export fn2()
```

is there a preference?

i notice that wrapping these funcs in `someServiceName` is helpful in avoiding name clashes. tanstack definition usually go like this:

```js
const fetchRole = .....tanstack lingo...  fetchRole // <--- standalone function 
// which now I need to rename it
// whereas if i wrapped it in an 
// object `roleService.fetchRole` no name clash
```

Reply:
I'd keep an eye on this but it's also something that's not terribly hard to refactor out of, unless the object starts being used for more than just a namespace for functions (e.g. we start adding some `this` calls or storing state on it).

FWIW, if you have top-level definitions of functions, it should be possible to later import them through `import * as foo` to have a similar effect as an object "namespace".